YogaKit/Source/YGLayout.m

```diff
@@ -304,10 +306,15 @@ static YGSize YGMeasureView(
   const CGFloat constrainedHeight = (heightMode == YGMeasureModeUndefined) ? CGFLOAT_MAX: height;
 
   UIView *view = (__bridge UIView*) YGNodeGetContext(node);
-  const CGSize sizeThatFits = [view sizeThatFits:(CGSize) {
-    .width = constrainedWidth,
-    .height = constrainedHeight,
-  }];
+  CGSize sizeThatFits = CGSizeZero;
+
+  // Fix for https://github.com/facebook/yoga/issues/606
```

Comment:
Instead of pointing to the issue. Can you comment in the code, why this check is added.

Reply:
Done.